File: xh_manager/src/main/webapp/js/service/abilityItemService.js
app.service("abilityItemService",function($http){
	
	//根据传递的模块儿id , 查询所有的基础选项数据
	this.findAllAbilityItemListByPart = function(part){
		return $http.get('../abilityItem/findAllAbilityItemListByPart.action?part='+part);
	};
	
	
	
	//查询所有的基础选项数据
	this.findAllOption = function(){
		return $http.get('../abilityItem/findAllOption.action');
	};
	
});

File: xh_manager/src/main/java/com/xh/service/AbilityItemService.java
package com.xh.service;

import java.util.List;
import java.util.Map;

import com.xh.pojo.AbilityItem;
import com.xh.pojo.TbAbilityItemOption;

public interface AbilityItemService {
	
	public List<AbilityItem> findAllAbilityItemListById(String part);
	
	
	public Map<Integer , Integer> getScoreLevelMapping();

	
	public List<TbAbilityItemOption> findAll();
	
}
